hotness/buildsys.py

```diff
@@ -112,33 +112,27 @@ def scratch_build(self, session, name, source):
             name=name, url=self.url_for(task_id)))
         return task_id
 
-    def run(self, cmd, cwd=None):
-        _log.info("Running %r in %r", cmd, cwd)
-        p = sp.Popen(cmd, cwd=cwd, stdout=sp.PIPE, stderr=sp.PIPE)
-        out, err = p.communicate()
-        if out:
-            _log.debug(out)
-        if err:
-            _log.warning(err)
```

Comment:
Bodhi had this pattern a lot too. A lot of programmers assume that stderr is used for errors, but it's actually quite often used for debug info. I've been trying to stop bodhi from treating stderr output as warnings or errors when I find things like this.